files/zh-cn/web/javascript/reference/statements/import.meta/index.html

```diff
@@ -73,7 +73,7 @@ <h2 id="规范">规范</h2>
   </tr>
   <tr>
    <td><a href="https://github.com/tc39/proposal-import-meta">tc39/proposal-import-meta</a></td>
-   <td>Stage 3</td>
+   <td>Stage 4</td>
```

Comment:
Please replace the whole `Spec` section with this macro `{{Specifications}}`.

Reply:
Fixed!